app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -0,0 +1,15 @@
+%fieldset
+  %legend
+    Add a councillor
+    = f.fields_for :suggested_councillors, @councillor_contribution.suggested_councillors.last do |suggested_councillor_field|
+      %p{:class => "nameField"}
+        = suggested_councillor_field.label :name
+        = suggested_councillor_field.text_field :name
+      %p{:class => "emailField"}
+        = suggested_councillor_field.label :email
+        = suggested_councillor_field.text_field :email
+  %p.button{:id => "addMoreCouncillor"}
```

Comment:
These buttons are indented bellow the `<fieldset>` element, but they actually aren't part of the [fieldset](https://html.spec.whatwg.org/multipage/form-elements.html#the-fieldset-element), which groups input elements for adding the councillors details.

I recon bump them up out of the fieldset.